torch/csrc/distributed/c10d/FlightRecorder.cpp

```diff
@@ -677,6 +695,8 @@ std::string FlightRecorder::dump(
   return pickle_str(result);
 }
 
+template class FlightRecorder<at::cuda::CUDAEvent>;
+
```

Comment:
Add a document here? i.e. if third party devices need to use FR, they/we would need to add
`template class FlightRecorder<c10::Event>;`
here.

Reply:
The c10::Event is added in later PR. I will comment in the later PR.